charts/partials/templates/_ext-proxy-admin-policy.tpl

```diff
@@ -0,0 +1,39 @@
+{{- define "partials.ext-proxy-admin-policy" -}}
```

Comment:
Note that we really don't need to have a separate admin server/authorization for each deployment. It should be sufficient to have a single admin server resource per namespace!

Reply:
Makes sense!